beacon-chain/types/active_state.go

```diff
@@ -197,18 +197,32 @@ func (a *ActiveState) CalculateNewActiveState(block *Block, cState *Crystallized
 	}, newBlockVoteCache), nil
 }
 
+func (a *ActiveState) getBlockHash(block *Block, slot uint64) ([32]byte, error) {
+	sback := block.SlotNumber() - (params.CycleLength * 2)
```

Comment:
what is `sback`?

Reply:
Nice to follow 2.1 spec but we shouldn't be treating it as a single source of truth, the spec still under reviewed and getting revamped. We should improve on it and provide feedback to the spec. In this case, I think `slotBack` sounds better